source/FilesServer/app/service.py

```diff
@@ -1,27 +1,25 @@
-from fastapi import HTTPException, status, Response
-from urllib.parse import quote
+from fastapi import HTTPException, status
+from fastapi.responses import RedirectResponse
 import boto3
 
 from app.settings import get_settings
 
 
-def get_bucket():  # TODO - add type hint
+def get_s3_client() -> boto3.client:
     settings = get_settings()
-    return boto3.resource("s3").Bucket(settings.aws_bucket_name)
+    return boto3.client("s3")
 
 
-async def s3_download(file_name: str) -> Response:
+async def s3_download(file_name: str) -> RedirectResponse:
```

Comment:
the service shouldn't return a RedirectResponse, just a string. the endpoint should wrap the url string in RedirectResponse